### DISCUSSION TITLE
Migration off IPTables to NFTables

### DISCUSSION THREAD

**Discussion by henryjb99** [6jarjar6/nordlynx#20] [Ideas]
As IPTables has been considered deprecated by most major distributions for 5 or more years, what do you think about this upgrading to the NFTables?

I know everytime my box reboots, I get warnings that IPTables will be fully removed from future versions. Here is a statement from Red Hat about IPTables:

https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/html/9.0_release_notes/deprecated_functionality#deprecated-functionality_networking

And notes about migrating away from IPTables to NFTables
https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/html/configuring_firewalls_and_packet_filters/getting-started-with-nftables_firewall-packet-filters?extIdCarryOver=true&sc_cid=701f2000001Css0AAC#assembly_migrating-from-iptables-to-nftables_getting-started-with-nftables

**Comment by 6jarjar6** [comment]
@henryjb99 
Please review my [PR](https://github.com/6jarjar6/nordlynx/pull/30)
And check the feature branch image out!

Apologies I don't have time to learn firewall rules so I did the half migration, by using IP table rules with the NF backend
